pillar/vector/mitx.sls

```diff
@@ -516,17 +507,23 @@ vector:
         index: logs-mitx-stderr-%Y.%W
         healthcheck: false
 
-      elasticsearch_tracking:
-        inputs:
-          - tracking_log_field_adder
-        type: elasticsearch
-        endpoint: 'http://operations-elasticsearch.query.consul:9200'
-        index: logs-mitx-tracking-%Y.%W
-        healthcheck: false
-
       elasticsearch_authlog:
         inputs:
           - auth_log_field_adder
         type: elasticsearch
         endpoint: 'http://operations-elasticsearch.query.consul:9200'
         index: logs-authlog-%Y.%W
+
+      s3_tracking:
+        inputs:
+          - tracking_log_timestamp_renamer
+        type: aws_s3
+        bucket: {{ salt.grains.get('environment') }}-tracking-logs
```

Comment:
We need to keep the S3 destination the same as it currently is, so they should be going to the buckets defined here: https://github.com/mitodl/salt-ops/blob/main/pillar/fluentd/server.sls#L6-L7

Reply:
OK, no problem. Would there by any issue with my changing the file naming scheme, or should I make sure the files are still named like `%Y-%m-%d-%H_[01].gz`? (Where `[01]` seems to be an identifier for the aggregator.)

The issue with that is we don't have Vector aggregators yet, and all the logs will be coming from many individual instances. If the format of the identifier doesn't matter and can be a UUID, this will be easy to implement.

Reply:
The identifier is arbitrary and doesn't matter so it should work fine with a UUID

Reply:
How is QA handled in the FluentD configuration above?

Reply:
It's not. For this setup it might be worth setting up a sampling rate to send ~5 or 10% of logs to S3 so we can smoke test it as new versions of Vector get deployed, etc.

Reply:
No, I meant send the QA tracking logs to a QA bucket, but only send ~5% of the log lines since we don't care as much about actually capturing that data.

Reply:
Resolved in Slack call: have separate buckets for QA and production, but don't worry about sampling QA, since it doesn't have a lot of traffic.